registrations/models.py

```diff
@@ -251,7 +251,40 @@ def fire_state_metric(sender, instance, created, **kwargs):
 
             total = get_or_incr_cache(
                 total_key,
-                registrations_for_state(state).count)
+                registrations_for_identity_field(
+                    {"details__state": state}).count)
```

Comment:
Probably a silly question: does this return the `count` or does it return a reference to the [`count()`](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.count) function?